testing/adios2/engine/bp/TestBPWriteProfilingJSON.cpp

```diff
@@ -40,47 +41,73 @@ class BPWriteProfilingJSONTest : public ::testing::Test
 // ADIOS2 write, native ADIOS1 read
 TEST_F(BPWriteProfilingJSONTest, ADIOS2BPWriteProfilingJSON)
 {
-    std::string fname = "ADIOS2BPWriteProfilingJSON.bp";
+    // Use a relative path + file name to test path in file name capability
+    std::string fname;
+    fname = "foo" + std::string(1, adios2::PathSeparator) + "ADIOS2BPWriteProfilingJSON.bp";
```

Comment:
@williamfgc I don't think we need the platform-dependent path separator.  The fopen calls should work correctly, even on windows, with foward-slash separators.

Reply:
Reference: https://msdn.microsoft.com/en-us/library/yeby3zcb.aspx